CHANGES.txt

```diff
@@ -1,3 +1,7 @@
+1.4.7 (2021-02-16)
+
+- Add basic_l1_all_frames asset type
+
```

Comment:
ah yes good call; let's put together a 1.4.7 release (separately from this PR) too